docs/start/quickstart.md

```diff
@@ -66,7 +66,7 @@ export default function App() {
 First build the app for production:
 
 ```shellscript nonumber
-remix build
+npx remix build
```

Comment:
Alternatively we could recommend adding these commands to the `package.json`'s `scripts`, but that seems like a lot more work.

If I'm missing something to get regular 'ol `remix build` to work on it's own, let me know! If so, it's likely something we should reference in this doc

Reply:
I think it's safe to use `npx` since w'ere using `npm` above.  `yarn`/`pnpm` users should know what to switch it to 👍

Reply:
This should work without adding `npx` once dependencies are installed
At least when using npm

Reply:
Huh?  How?

```sh
~/me/shopify/repos/playgrounds/remix-playground> remix build
-bash: remix: command not found
```

Reply:
`remix` & `remix-serve` bins are installed, so they should be available to call normally 🤔

Reply:
`bin` scripts still require `npx` when installed locally. If installed globally, those scripts are added to the path and can be called like any other command.

Reply:
☝️ 

pretty much the reason `npx` was created from my understanding

Reply:
oh phew I thought I missed an `npm` behavior change recently - I've given up on keeping up with package manager evolutions 🤣

Reply:
I just looked for about 2h into why this was working for me and found the culprit 😅
https://github.com/MichaelDeBoey/dotfiles/blob/0e92b1d9b524f100ca8ec574f971c5f18154c7d0/path.zsh#L8